### PR TITLE
worked on penington rigge branson

### PR DESCRIPTION
**Pull request by Henderjay**
worked on penington rigge and branson

### CODE REVIEW

File: src/en/isaac-penington.yml [hunk 1]
```diff
@@ -25,6 +25,181 @@ documents:
...
+          parts:
+            -
+              title: Part 1
+              external_id_hq: 236087816
+              external_id_lq: 236087828
+              filesize_hq: 48529714
+              filesize_lq: 17356902
+              seconds: 2893
+              chapters:
+                - 1
```
Comment (jaredh159) [inline src/en/isaac-penington.yml]: These should start with `0` if it corresponds to the very first chapter.  Take a look at the `william-penn.yml`.

File: src/en/isaac-penington.yml [hunk 2]
```diff
@@ -99,6 +274,172 @@ documents:
...
+          parts:
+            -
+              title: Part 1
+              external_id_hq: 251385524
+              external_id_lq: 251385543
+              filesize_hq: 68549849
+              filesize_lq: 24184446
+              seconds: 4031
+              chapters:
+                - 1
```
Comment (jaredh159) [inline src/en/isaac-penington.yml]: same here, start from `0` and go up

File: src/es/isaac-penington.yml [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+            type: pdf
+          -
+            type: epub
+          -
+            type: mobi
+        audio:
+          reader: Jason Henderson
+          parts:
+            -
+              title: Part 1 # All of these audio parts are for the ENGLISH audiobook, and need to be redone for Spanish.
```
Comment (jaredh159) [inline src/es/isaac-penington.yml]: Do the spanish recordings exist?  Are they on Soundcloud?  I'm wondering if we should either a) get the real recordings dialed in here (I can explain how I think), or b) remove this until we're able to put them here....

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Cool, most of this looks really good. I left a few comments as a review with a couple things to change and one thing I had a question about.

This is the first PR that I've left a review for, because at least one thing needs to be changed. So, what you'll want to do is:

a) you can comment back to my comments if we need to discuss any of my questions or proposed changes

b) once you know what changes you're going to make, check out this branch again locally and make the changes.  Then make a new commit and push back up `git push jason BRANCH_NAME`.  Then I'll review the changes and leave more comments or approve and merge.

**Comment by Henderjay** [comment]
I think I made the necessary changes.  Killed the spanish audio, started chapter numbers with 0, and changed chapter titles to Chapter 1, 2, 3, etc. since they are not really "parts" in this book.

**Comment by jaredh159** [comment]
@Henderjay looks good. merged. 👍

**Pull request merged**